src/suit/js/SUIT.js

```diff
@@ -30,25 +31,26 @@ var props = {
     ],
 };
 
+function findCorrectTapService() {
+    const url= window.location.href;
+    const idx= url.indexOf('/portal');
+    if (idx===-1) return;
+    return `${url.substr(0,idx)}/api/tap`;
+}
+
 var options = {
     MenuItemKeys: {maskOverlay: true},
     imageTabs: ['fileUpload', 'url', '2mass', 'wise', 'sdss', 'msx', 'dss', 'iras'],
     irsaCatalogFilter: 'lsstFilter',
     catalogSpacialOp: 'polygonWhenPlotExist',
-    // hips : {useForCoverage: false, useForImageSearch: true,
-    //     hipsSources: 'all',
-    //     defHipsSources: {source: 'irsa', label: 'Featured'},
-    //     mergedListPriority: 'irsa'},
     tap : {
         services: [
-            { label: 'LSST lsp-stable https://lsst-lsp-stable.ncsa.illinois.edu/api/tap',
-                value: 'https://lsst-lsp-stable.ncsa.illinois.edu/api/tap' },
-            { label: 'LSST lsp-int https://lsst-lsp-int.ncsa.illinois.edu/api/tap',
-                value: 'https://lsst-lsp-int.ncsa.illinois.edu/api/tap' },
             { label: 'CADC https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/tap',
                 value: 'https://www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca/tap' },
             { label: 'GAIA https://gea.esac.esa.int/tap-server/tap',
                 value: 'https://gea.esac.esa.int/tap-server/tap' },
+            { label: 'HSA https://archives.esac.esa.int/hsa/whsa-tap-server/tap',
```

Comment:
They were all in alphabetical order before, I just realized.  Can you move HSA down one step, below GAVO?